DESCRIPTION

```diff
@@ -2,26 +2,26 @@ Package: dbpath
 Title: What the Package Does (One Line, Title Case)
 Version: 0.0.0.9000
 Authors@R: 
-    person(given = "Michael",
-           family = "Chow",
-           role = c("aut", "cre"),
-           email = "mc_al_github@fastmail.com")
+    person("Michael", "Chow", , "mc_al_github@fastmail.com", role = c("aut", "cre"))
 Description: A url interface for connecting to databases.
 License: MIT + file LICENSE
-Encoding: UTF-8
-LazyData: true
-Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.1
 Imports: 
-    dplyr,
     DBI,
+    dplyr,
     methods
 Suggests: 
-    testthat,
-    RPostgres,
+    odbc,
     RMariaDB,
-    odbc
+    RPostgres,
+    testthat
+Config/testthat/edition: 3
```

Comment:
Turned on testthat 3rd edition, which required one change elsewhere, but lets us use `expect_snapshot()` to test the dbpath print method.